docs/usage/querying.rst

```diff
@@ -154,7 +154,17 @@ these parameters by the method `custom(name: str, value: str)`.
 
     employees = northwind.entity_sets.Employees.get_entities().custom('sap-client', '100').custom('$skiptoken', 'ABCD').top(10).execute() 
     
+Encode OData URL Path
+-------------------------------------------
+
+Sometimes services expect the path to be percent encoded. This can especially be important 
```

Comment:
delete the "especially" word. Just for readability, "especially" followed by "special variable" is unnecessary.